framework/include/meshmodifiers/MeshExtruder.h

```diff
@@ -31,8 +31,13 @@ class MeshExtruder : public MeshModifier
 protected:
   virtual void modify() override;
 
-  const unsigned int _num_layers;
   const RealVectorValue _extrusion_vector;
+  unsigned int _num_layers;
+
+  bool _map_custom_ids;
+  std::vector<SubdomainID> _existing_subdomains;
```

Comment:
These variables are created so that I can manipulate them before call `MeshExtruder::modify()`.